phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/MultiThreadedRunner.java

```diff
@@ -131,14 +142,16 @@ private void timedQuery() throws Exception {
         Long start = EnvironmentEdgeManager.currentTimeMillis();
```

Comment:
Based on our offline discussion, let's consolidate the start and end times for finding query elapsed time. We already define a `start` time here and also do the same inside `getResults`. Also the final elapsed time that's written to the csv is not consistent with the one we check against the query timeout